recipes/hostpython3/__init__.py

```diff
@@ -28,33 +28,11 @@ def prebuild_arch(self, arch):
         if self.has_marker("patched"):
             return
         self.copy_file("ModulesSetup", "Modules/Setup.local")
-        # self.apply_patch("ssize-t-max.patch")
-        # self.apply_patch("dynload.patch")
-        # self.apply_patch("static-_sqlite3.patch")
-        # shutil.copy("Modules/Setup.dist", "Modules/Setup")
-        # if "openssl.build_all" in self.ctx.state:
-        #     self.append_file("ModulesSetup.openssl", "Modules/Setup.local")
         self.set_marker("patched")
 
     def postbuild_arch(self, arch):
         return
```

Comment:
Same here with the dead code and commented lines, how about dropping it if we don't use?

Reply:
Same as before

Reply:
Yes never mind, I brain farted big time :smile: